src/App.tsx

```diff
@@ -21,14 +34,17 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length
+                ? <Loader />
```

Comment:
Hm🤔 Let's think what will be if there is an empty array of todos from the server. You should add a state to check if it's loading or not